docs/tls-disable.md

```diff
@@ -0,0 +1,144 @@
+# Run Percona Server for MongoDB without TLS
+
+You can run Percona Server for MongoDB without TLS. For example, for testing or demonstration purposes. However, we recommend that you have the TLS protocol enabled.
+
+You can start a new cluster without TLS or disable the TLS protocol for a running cluster. See the corresponding sections for steps.
+
+## Disable TLS for a new cluster
+
+To disable TLS protocol for a new cluster, edit the `deploy/cr.yaml` Custom Resource manifest as follows:
+
+* set the `tls.mode` key to `disabled` 
+* set the `unsafeFlags.tls` to `true`.
+
+```yaml
+...
+spec:
+  ...
+  unsafeFlags
+    tls: true
+    ...
+  tls:
+    mode: disabled
+```
+
+Apply the manifest:
+
+```{.bash data-prompt="$" }
+$ kubectl apply -f deploy/cr.yaml -n <namespace>
+```
+
+## Disable TLS for a running cluster
+
+To disable TLS protocol for a running cluster, follow these steps:
+
+1. Pause the cluster. Edit the `deploy/cr.yaml` Custom Resource manifest and set `spec.pause` key to `true`:
+
+    ```yaml
+    spec:
+      pause: true
+    ```
+
+2. Apply the changes:
+
+    ```{.bash data-prompt="$" }
+    $ kubectl apply -f deploy/cr.yaml -n <namespace>
```

Comment:
maybe it is better to use the patch command 

Pause the cluster
kubect patch pxc ${cluster_name} --type json -p='[{"op":"add","path":"/spec/pause","value":true}]'

Unpause the cluster
kubectl patch pxc ${cluster_name} --type json -p='[{"op":"add","path":"/spec/pause","value":false}]'

Reply:
Do we also need to use kubectl patch for enable/disable TLS?